### DISCUSSION TITLE
Version 3.6.4 is available.

### DISCUSSION THREAD

**Discussion by janiversen** [pymodbus-dev/pymodbus#1970] [Announcements]
The release contains a number of important bug fixes, thanks to a number of new (and old) volunteers.

Thanks to everybody who helped this release come through.

In the background we are working on 3.7.0, which hopefully will reduce the code complexity a lot and have a new configuration style for the simulator. V3.7.0 is not to be expected before late spring.

**Comment by MAKOMO** [comment]
Thanks for all your excellent work on pymodbus. Very appreciated!
